openshift_tools/monitoring/dockerutil.py

```diff
@@ -196,3 +197,57 @@ def get_ctr_stats(self, ctr, use_cgroups=False):
             raw_stats = self._docker.stats(ctr['Id'], stream=False)
 
         return CgroupUtil.raw_stats_to_dtos(raw_stats)
+
+class CleanupDockerStorage(object):
+    """clean up the docker storage data and metadata"""
+    def __init__(self, docker_client=None):
+        'initial of the client'
+        self.client = docker_client
+        self.all_exited_con_info = None
+        self.all_dangling_img_info = None
+
+        self.exited_con = []
+        self.useless_img = []
+
+    def get_exited_cons(self):
+        'get the exited containers list'
+        self.all_exited_con_info = self.client.containers(all='true', filters={'status': 'exited'})
+        for container in self.all_exited_con_info:
+            self.exited_con.append(container['Id'])
+        return self.exited_con
```

Comment:
Do you really need to return self.exited_con? The only caller of this method is CleanupDockerStorage.run() and it already has access to self.exited_con.